lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -2607,6 +2607,72 @@ def visit_computed_column(self, generated):
             text += " PERSISTED"
         return text
 
+    def visit_set_table_comment(self, create):
+        schema = self.preparer.schema_for_object(create.element)
+        schema_name = schema if schema else self.dialect.default_schema_name
+        return """
+           execute sp_addextendedproperty 'MS_Description', {0}, 'schema', {1}, 'table', {2};
+           """.format(
+            self.sql_compiler.render_literal_value(
+                create.element.comment, sqltypes.String()
+            ),
+            quoted_name(schema_name, quote=True),
+            quoted_name(
+                self.preparer.format_table(create.element, use_schema=False),
```

Comment:
here I think you can use return value of format table directly?

Reply:
done, it was something I added to try to get past the QuotedNameArgumentTest unit tests